contents/overlay/dialogs/simple-dialog/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { Trash2 } from "@yamada-ui/lucide"
+import {
+  Button,
+  Dialog,
+  DialogHeader,
+  Text,
+  useDisclosure,
+  VStack,
+} from "@yamada-ui/react"
+import type { FC } from "react"
+
+const SimpleDialog: FC = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure()
+
+  return (
+    <>
+      <Button onClick={onOpen}>Open Dialog</Button>
+      <Dialog
+        isOpen={isOpen}
+        onClose={onClose}
+        cancel="Cancel"
+        onCancel={onClose}
+        success="Delete"
```

Comment:
```suggestion
        success={{
          colorScheme: "danger",
          children: "Delete",
        }}
```

If the delete button is meant to be red, I think it should be red.

Reply:
Applied it!

Reply:
Thanks!